Modules/System/Azure File Services API/src/AFSOperationResponse.Codeunit.al

```diff
@@ -0,0 +1,84 @@
+/// <summary>
```

Comment:
Consolidating this and the Azure Blob Storage module would be good. As these are basically duplicate code, probably consider a separate module for the commonality between the storage modules. We may get more in the future.

Reply:
Okay, I will check out what can be merged into common module and let you know here.